src/components/MyAccount/Settings/PasswordModal.tsx

```diff
@@ -46,11 +47,7 @@ const PasswordModal = ({ patron }: { patron: Patron }) => {
       </Box>
     ),
     closeButtonLabel: "Cancel",
-    headingText: (
-      <Heading className={styles.modalHeading}>
-        <Text sx={{ marginBottom: 0 }}> Change PIN/PASSWORD </Text>
-      </Heading>
-    ),
+    headingText: modalHeading(<Text> Change PIN/PASSWORD </Text>),
```

Comment:
Not sure if this is a part of this PR or the CSS one, but remove the `Text` wrapper.